nx-X11/programs/Xserver/mi/miinitext.c

```diff
@@ -228,6 +224,11 @@ extern void SecurityExtensionInit(void);
 extern void XFree86BigfontExtensionInit(void);
 #endif
 #ifdef GLXEXT
+/*
+typedef struct __GLXprovider __GLXprovider;
+extern __GLXprovider __glXMesaProvider;
+extern void GlxPushProvider(__GLXprovider *impl);
```

Comment:
Hm, the original code has these lines non-commented-out.

They seem to stem from the original AIGLX inclusion... I guess we won't actually need this in nxagent and it might not be actually working with the built-in software renderer, but we should probably still keep this in mind.